src/base/rss/rssdownloadrule.h

```diff
@@ -88,6 +88,9 @@ namespace Rss
         bool operator==(const DownloadRule &other) const;
 
     private:
+        bool matches(const QString &articleTitle, QString expression) const;
```

Comment:
Needs to be QString because I call expression.split() (yes, seems weird, but that's how it is).